### DISCUSSION TITLE
Changing scroll behavior on column or row header select

### DISCUSSION THREAD

**Discussion by chuckurbis** [nick-keller/react-datasheet-grid#122] [Feature requests]
When clicking a column heading the table is scrolling to the end, and similarly when I selected a row number it scrolls to the far right end of that row. 

Is this behavior controllable? Ideally it would stay put there.

**Comment by nick-keller** [comment]
Agreed, this is annoying, I'll try to fix this soon!

**Comment by nick-keller** [comment]
This is now live in 4.1.0

**Reply by chuckurbis** [reply]
You rock.
